spt/OrangeBox/overlay/overlays.cpp

```diff
@@ -27,19 +27,21 @@ CameraInformation rearViewMirrorOverlay()
 CameraInformation havokViewMirrorOverlay()
 {
 	CameraInformation info;
-	Vector add(0, 0, 64);
-	Vector* pos = &(vphysicsDLL.PlayerHavokPos + add);
+	Vector* havokpos = &(vphysicsDLL.PlayerHavokPos);
```

Comment:
```suggestion
	Vector havokpos = vphysicsDLL.PlayerHavokPos;
```
tbh i'd just copy here